AllReadyApp/Web-App/AllReady/DataAccess/AllReadyDataAccessEF7.Activity.cs

```diff
@@ -20,6 +20,7 @@ IEnumerable<Activity> IAllReadyDataAccess.Activities
                                 .Include(a => a.Tasks)
                                 .Include(a => a.RequiredSkills)
                                 .Include(a => a.UsersSignedUp)
+                                .Where(a => !a.Campaign.Locked)

```

Comment:
I realize this only has one referrer in `ActivityApiController` but it doesn't feel quite right to exclude locked activities _before_ returning from the repository class. I'd rather see those locked activities filtered out in `ActivityApiController`.